packages/spec/integration/graphql-mock-server/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "fixture-graphql",
+  "version": "1.0.0",
+  "private": true,
+  "engines": {
+    "node": ">= 8.10"
+  },
+  "hops": {
+    "shouldPrefetchOnServer": false,
+    "port": "8029",
+    "graphqlUri": "http://localhost:8029/graphql",
+    "fragmentsFile": "<rootDir>/fragmentTypes.json",
```

Comment:
Isn't this the default and therefore doesn't need to be set?

Reply:
`fragmentsFile` is by default set to `<rootDir>/fragmentTypes.json`: https://github.com/xing/hops/blob/master/packages/graphql/preset.js#L2
`shouldPrefetchOnServer` is on by default, I think.